bertopic/representation/_llamacpp.py

```diff
@@ -93,14 +114,15 @@ def __init__(
         self,
         model: Union[str, Llama],
         prompt: str = None,
+        system_prompt: str = None,
         pipeline_kwargs: Mapping[str, Any] = {},
         nr_docs: int = 4,
         diversity: float = None,
-        doc_length: int = None,
+        doc_length: int = 100,
         tokenizer: Union[str, Callable] = None,
     ):
         if isinstance(model, str):
-            self.model = Llama(model_path=model, n_gpu_layers=-1, stop="Q:")
+            self.model = Llama(model_path=model, n_gpu_layers=-1, stop="\n", chat_format="llama-2")
```

Comment:
What makes you choose `llama-2` as the default chat template? Should we choose one at all or are that perhaps more common templates like `ChatML`?

Reply:
I followed the readme in llamacpp, I have changed it to `ChatML`